core/src/main/scala/org/apache/spark/sql/delta/catalog/DeltaCatalog.scala

```diff
@@ -188,6 +192,40 @@ class DeltaCatalog extends DelegatingCatalogExtension
     }
   }
 
+  override def loadTable(ident: Identifier, timestamp: Long): Table = {
+    loadTableWithTimetravel(ident, version = None, Some(timestamp))
+  }
+
+  override def loadTable(ident: Identifier, version: String): Table = {
+    loadTableWithTimetravel(ident, Some(version), timestamp = None)
+  }
+
+  /**
+   * Helper method which loads a Delta table with given time travel parameters.
+   * Only one of the timetravel inputs (version or timestamp) be present.
+   * @return
+   */
+  private def loadTableWithTimetravel(
+      ident: Identifier,
+      version: Option[String],
+      timestamp: Option[Long]): Table = {
+    val table = loadTable(ident)
+    assert((version.isDefined && timestamp.isEmpty) || (version.isEmpty && timestamp.isDefined),
+      "Expected the timetravel parameters consists of only one: either the timestamp or version")
```

Comment:
Why not use `DeltaTimeTravelSpec` and borrow [this lines](https://github.com/delta-io/delta/blob/81d56794b1ea1032b1d7726c343325661e8ed20f/core/src/main/scala/org/apache/spark/sql/delta/DeltaTimeTravelSpec.scala#L43-L44):

```
  assert(version.isEmpty ^ timestamp.isEmpty,
    "Either the version or timestamp should be provided for time travel")
```

Reply:
Good idea, sounds good.